WordPressKit/State.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+@objc class State: NSObject, Codable {
```

Comment:
A one-line comment explaining what we're using this for would be good — when I think of `State` in the context of programming, this isn't the first thing that comes to mind :)

Reply:
(A similar comment could be added to `Country.swift`, however there the possibility of confusion is much smaller :))